### PR TITLE
Set up "Hello Microverse" project

### PR DESCRIPTION
**Pull request by developertariq**
Create branches with different base branches.
Understand the concept of version control.
Follow GitHub flow.
Open pull requests in GitHub.
Write descriptive, easy-to-understand commit messages.
Write short, detailed, easy-to-understand descriptions of pull requests.
Write a descriptive and easy-to-understand README in business English for every project submitted.
Demonstrate an ability to submit a project for code review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+## Hello-Microverse
+
+### This is my first project at Microverse. In this project will cover the following issues,
+
+Working with different branches.
+Using linters in code base.
+Add .gitignore file.
+Document a repo in a professional way.
+Write a descriptive and understandable README file.
+Submit project for reviews.
```
Comment (mckent05) [inline README.md:10]: - It is very important you write a well-documented readme in order to make your repo look more professional. I'd suggest you use this [readme](https://github.com/microverseinc/readme-template/blob/master/README.md) template from Microverse 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @developertariq,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- Your PR title and summary are descriptive 👍 👍 
- Correct GitHub flow is used 👍 👍 


## Required Changes ♻️
- Some of your linter checks aren't passing, I'd suggest you check for the errors and try to fix them 🙏 . Linters help us to ensure that we write code that is clean and maintainable.
- According to the project requirements, your `.gitignore` should contain `test.md` and `node_modules`, please try to fix this 🙏 
- According to the project requirements your `index.html` file should have an `h1` tag with "Hello Microverse"  written in it and should be styled with your favorite color in your `stles.css` file.Please try to fix this 👍 👍 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (developertariq) [reply README.md:10]: After a long time, I solved this problem according to the requirements. Please review the solutions.

Comment (developertariq) [reply README.md:10]: @mckent05 After a long time, I solved this problem according to the requirements. Please review the solutions.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 Hi there  @developertariq !
 
You have successfully completed this project! There is nothing else to say other than... it's time to merge it! rocket Time to celebrate, you have done a fantastic job, congratulations! :tada:

  
 ##  Highlights 
 * Great work using the right GitHub Flow 100
 * Awesome file organization card_file_box
 * Commits are descriptive, good work.
 * Linters are working properly, great job!

 APPROVED! :100: :100: :100:

## Optional suggestions 
 _Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 100_
 
 * [ ]  **[OPTIONAL]: :point_down: 

- [ ] You  did great work on this project , it will be much more better if your readme looks professional, I made this as an option because it was required in the previous review but not really satisfied,It is very important you write a well-documented readme in order to make your repo look more professional. I'd suggest you use this [readme](https://github.com/microverseinc/readme-template/blob/master/README.md) template from Microverse :wink: 

Cheers and best of luck with the following projects! :raised_hands:
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me **@alfredbis29** in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**